test/trae/instance.spec.ts

```diff
@@ -6,13 +6,69 @@ import trae from '../../src';
 describe('Trae', function () { 
   it('exposes a function to create a Trae instance', () => {
     const actual = typeof trae
+    const expected = 'object'
```

Comment:
I don't think the tests in this file are relevant anymore. Most of it should be covered by types and indirectly by other tests.

Eg: the tests of `trae.get` cover the fact that `trae.get` should be there and should be a function.

Reply:
Makes sense, I just left a test to make sure we are exposing what we actually want.